section-05-production-model-package/setup.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Comment:
I thought we would cover python package in a later section after this one and the API.

So in my head:
section 5: writing production code
section 6: FastAPI
section 7: packaging.

If we are still onboard for this one, I would remove files that are not needed in this section and ship them to section 7. There are already a lot of scripts to go over with the students